src/components/DynamicFormContainer/DocumentSelector/DocumentSelector.tsx

```diff
@@ -1,49 +1,56 @@
+  validateCurrentForm,
+  closePreviewMode,
+}) => {
+  const { forms, setActiveFormIndex, activeFormIndex } = useFormsContext();
 
-  const previousDocument = (): void => {
-    if (activeFormIndex === undefined || activeFormIndex === 0) return;
-    if (validateCurrentForm()) setActiveFormIndex(activeFormIndex - 1);
+  const selectDocument = (formIndex: number): void => {
+    if (isNaN(formIndex)) return;
+    if (activeFormIndex === undefined || formIndex > forms.length) return;
+    if (validateCurrentForm()) setActiveFormIndex(formIndex - 1);
     closePreviewMode();
   };
 
-  const nextDocument = (): void => {
-    if (activeFormIndex === undefined || activeFormIndex === forms.length - 1) return;
-    if (validateCurrentForm()) setActiveFormIndex(activeFormIndex + 1);
-    closePreviewMode();
-  };
+  const activeDocumentNumber = activeFormIndex ? activeFormIndex + 1 : 1;
+
   return (
-    <div className="flex items-center mt-2">
-      <div
-        data-testid="previous-document-button"
-        onClick={previousDocument}
-        className="cursor-pointer border border-solid border-gray-300 h-12 w-12 bg-gray-100 active:bg-gray-400 border-r-0 flex items-center justify-center"
-      >
-        <ChevronLeft className="flex items-center justify-center" />
+    <div>
+      <div className="flex items-center">
+        <input
+          data-testid="document-number-input"
+          value={activeDocumentNumber}
+          onChange={(e) => {
+            selectDocument(parseInt(e.target.value));
+          }}
+          type="text"
+          className="flex rounded border border-solid border-gray-300 h-10 w-10 text-center"
+        />
+        <div>&nbsp;of {forms.length} document(s)</div>
       </div>
-      <input
-        data-testid="file-name-input"
-        type="text"
-        aria-label="file-name-input"
-        value={currentForm?.fileName}
-        onChange={(e) => setCurrentFileName(e.target.value)}
-        className="h-12 border border-solid border-gray-300 px-3 text-gray-800 text-lg"
-      />
-      <div
-        data-testid="next-document-button"
-        onClick={nextDocument}
-        className="cursor-pointer border border-solid border-gray-300 h-12 w-12 bg-gray-100 active:bg-gray-400 border-l-0 flex items-center justify-center"
-      >
-        <ChevronRight className="flex items-center justify-center" />
+      <div>
+        <select
+          data-testid="document-name-select"
+          value={activeDocumentNumber}
+          className="border border-cloud-200 rounded p-2 mt-3 min-w-full"
+          onChange={(e) => selectDocument(parseInt(e.target.value))}
+        >
+          {forms.map((form, formIndex) => {
+            const documentNumber = formIndex + 1;
+            return (
+              <option key={`${form.fileName}-${documentNumber}`} value={documentNumber}>
+                {form.fileName}
+              </option>
+            );
+          })}
+        </select>
```

Comment:
able to make the dropdown below the input field?

Reply:
isn't it already below the input field?

Reply:
On my end it's not below, it's on the input field. Can send me a screen shot of urs?

Reply:
![image](https://user-images.githubusercontent.com/34233605/132320922-2471d983-1a43-45d4-8dbc-27254d8b910e.png)

Reply:
### Actual
![image](https://user-images.githubusercontent.com/34233605/132321017-74c3d9bc-9d1f-4686-a70c-76d1fce291b7.png)

### Expected (Figma)
![image](https://user-images.githubusercontent.com/34233605/132321141-0bc63399-1305-46d8-8cfa-9fe05197a11d.png)